src/providers/DataSyncProvider.tsx

```diff
@@ -111,8 +111,10 @@ const DataSyncProvider: FC<{ children: ReactNode }> = ({ children }) => {
 
   useEffect(() => {
     if (!status) {
```

Comment:
If I didn't make this change, the status retrieval was getting stuck because there was no new interval called and therefor the status wasn't updating and the `useEffect` dependencies never changed.